lib/smart_answer_flows/marriage-abroad/outcomes/outcome_same_sex_for_non_romanians.govspeak.erb

```diff
@@ -39,13 +39,7 @@
     <%= render partial: 'partner_naturalisation_in_uk.govspeak.erb' %>
   <% end %>
 
-  <%= render partial: 'consular_fees_table.govspeak.erb',
-             locals: { calculator: calculator } %>
-
-  <%= render partial: 'link_to_consular_fees.govspeak.erb',
-             locals: { calculator: calculator } %>
-
-  <%= render partial: 'payment_information.govspeak.erb',
+  <%= render partial: 'os_affirmation/fees.govspeak.erb',
```

Comment:
I've added a [Trello card to remind us to rename this outcome](https://trello.com/c/XG0foVvN). I've also updated this commit note to highlight the confusingly named outcome template.